views/sessions/new.hbs

```diff
@@ -1,16 +1,18 @@
 {{#if error}}
-  <div class="error">{{error}}</div>
+  <p class="error">{{error}}</p>
 {{/if}}
 
- <h1 class="text-center mb-4">Sign In</h1>
+<h1>Sign In</h1>
+<div class="container">
 <form id="new-session-form" action="/sessions" method="post" class="p-5">
-  <div class="form-group">
+  <div class="item form-group">
     <label for="email">Email:</label>
     <input type="email" class="form-control" id="email" name="email" value="" placeholder="Enter email" required>
   </div>
-  <div class="form-group">
+  <div class="item form-group">
```

Comment:
welcome to use old class form-group, I feel like it's more meaningful than item. Or change to form-item?